utils/check_repo.py

```diff
@@ -257,6 +270,9 @@
     "AltCLIPTextModel",
     "AltCLIPVisionModel",
     "AltRobertaModel",
+    "SpeechT5ForSpeechToSpeech",
```

Comment:
Makes sense to me to add auto classes for "Speech-to-Speech" and "Text-to-Speech" in this PR